install_files/ansible-base/roles/upgrade/tasks/0-3pre-upgrade.yml

```diff
@@ -0,0 +1,21 @@
+---
+# This role will upgrade 0.3pre instances to 0.3 instances.
+# It will copy and run the upgrade.py script on the app and monitor servers.
+- name: remove old 0.3 packages to avoid version error
+  apt: name={{ item }} state=absent
+  with_items:
+    - securedrop-app-code
+    - securedrop-grsec
+    - securedrop-ossec-agent
+    - securedrop-ossec-server
```

Comment:
correct, ansible will not throw an error because it just checks that it is absent. So it does work on both servers.